options/posix/include/mlibc/posix-sysdeps.hpp

```diff
@@ -166,7 +168,7 @@ int sys_vm_unmap(void *pointer, size_t size);
 // as an infinite timeout, making sigtimedwait(..., nullptr) equivalent to sigwaitinfo(...)
 [[gnu::weak]] int sys_sigtimedwait(const sigset_t *__restrict set, siginfo_t *__restrict info, const struct timespec *__restrict timeout, int *out_signal);
 [[gnu::weak]] int sys_kill(int, int);
-[[gnu::weak]] int sys_accept(int fd, int *newfd, struct sockaddr *addr_ptr, socklen_t *addr_length);
+[[gnu::weak]] int sys_accept(int fd, int *newfd, struct sockaddr *addr_ptr, socklen_t *addr_length, int flags);
```

Comment:
The change of this sysdep should be propagated to other ports as well. Also make sure that the functionality stays the same (so copy the old ``fcntl()`` trick to the sysdeps as well).